R/later.R

```diff
@@ -15,6 +14,14 @@
 # this registry to keep the loop objects alive.
 .loops <- new.env(parent = emptyenv())
 
+new_weakref <- function(loop) {
+  .Call(`_later_new_weakref`, loop)
+}
```

Comment:
Please add a comment explaining why we have our own implementations instead of using `rlang::new_weakref`.

Reply:
This is also added in the same commit.